hack/push-and-tag.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+# File containing the list of images
+OUTPUT_TASK_BUNDLE_LIST="${OUTPUT_TASK_BUNDLE_LIST-${SCRIPTDIR}/../task-bundle-list}"
+
+# Read the file and process each line
+while IFS=, read -r original_image new_image; do
```

Comment:
I'm a bit confused about setting IFS to `,`, should it be `""`, and why have the `new_image` on this line?

Reply:
I'm [seeing it](https://github.com/konflux-ci/build-definitions/pull/1167/files#diff-06250a838abc96be6c8e0b4d81d7eab197b36fbb32a775795ef0c6e68a40ac9eR24) now, for me this needed to be in the `hack/build-and-push.sh` via `save_ref` function; this can work too.